### PR TITLE
chore(Dockerfile): cleanup after installation a bit more than before and keep copyright / license files

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

**Review by jeroenvisser101 — APPROVED** [review]

File: rootfs/Dockerfile [hunk 1]
```diff
@@ -8,8 +8,25 @@ RUN addgroup --quiet --gid 2000 slug && \
 RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
     apt-get update && \
     apt-get install -y md5deep && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
```
Comment (kmala) [inline rootfs/Dockerfile:12]: i don't see /tmp/\* and /var/tmp/\* removed below

Comment (helgi) [reply rootfs/Dockerfile:12]: Ah good point - may have to redo all my PRs

Comment (helgi) [reply rootfs/Dockerfile:12]: @kmala having looked at other images, that always seems to be empty even without that removal - is there anything we place in /tmp or /var/tmp ourselves?

Comment (kmala) [reply rootfs/Dockerfile:12]: we use /tmp during the slug building process https://github.com/deis/slugbuilder/blob/master/rootfs/builder/build.sh#L14-L18

Comment (helgi) [reply rootfs/Dockerfile:12]: @kmala ah right but that is when we build slugs, not when we build the image for the slugbuilder itself, right?

Comment (kmala) [reply rootfs/Dockerfile:12]: yes.

**Pull request merged**